instamojo_wrapper/api.py

```diff
@@ -29,8 +29,25 @@ def auth(self, username, password):
         else:
             raise Exception(response['message'])  # TODO: set custom exception?
 
-    def links_list(self):
-        response = self._api_call(method='get', path='links/')
+    def links_list(
+            self,
+            limit=None,
+            page=None,
+    ):
+
+        path = 'links/'
+
+        query_dict = dict(
+            limit=limit,
+            page=page,
+        )
+
+        query_string = urlencode(dict((k, v) for k, v in query_dict.items() if v is not None))
+        if query_string:
+            path += '?' + query_string
+
+        print(path)
```

Comment:
can we remove this?

Reply:
The print statement, right?
I'll do it asap